components/builder-api-proxy/habitat/default.toml

```diff
@@ -1,57 +1,58 @@
-app_url                 = "https://bldr.habitat.sh"
-cookie_domain           = ""
-demo_app_url            = "https://www.habitat.sh/demo/build-system/steps/1/"
-docs_url                = "https://www.habitat.sh/docs"
-enable_publisher_amazon = false
-enable_publisher_azure  = false
-enable_publisher_docker = false
-enable_builder          = false
-environment             = "production"
-hosted                  = false
-source_code_url         = "https://github.com/habitat-sh/habitat"
-status_url              = "https://status.habitat.sh/"
-tutorials_url           = "https://www.habitat.sh/learn"
-use_gravatar            = true
-www_url                 = "https://www.habitat.sh"
+app_url                   = "https://bldr.habitat.sh"
```

Comment:
Convert over the tabs back to spaces in the toml files?

Reply:
I used a plugin to do the alignment.  I had no idea it used tabs instead of spaces.  How can you tell the difference from the diff?

Reply:
I went into this file in my editor, and I can arrow through the spaces one by one, as well as traverse the lines vertically, maintaining the same column position, at any point on any of these lines.  They don't behave like tabs.

Reply:
I further configured my editor to show the ASCII character code of the character under the cursor, and ran through all these lines, and indeed, they are spaces, not tabs.

Reply:
ah ok cool.. for some reason I assumed the bold whitespace was showing tabs, but looks like that is not the case